packages/fxa-auth-server/lib/routes/password.js

```diff
@@ -395,28 +406,35 @@ module.exports = function (
           }
         }
 
-        function createKeyFetchToken() {
+        async function createKeyFetchToken() {
           if (wantsKeys) {
             // Create a verified keyFetchToken. This is deliberately verified because we don't
             // want to perform an email confirmation loop.
-            return db
-              .createKeyFetchToken({
+            if (authPW) {
+              keyFetchToken = await db.createKeyFetchToken({
                 uid: account.uid,
                 kA: account.kA,
                 wrapKb: wrapKb,
                 emailVerified: account.emailVerified,
-              })
-              .then((result) => {
-                keyFetchToken = result;
               });
+            }
+
+            if (authPWVersion2) {
+              keyFetchToken2 = await db.createKeyFetchToken({
+                uid: account.uid,
+                kA: account.kA,
+                wrapKb: wrapKbVersion2,
+                emailVerified: account.emailVerified,
+              });
+            }
           }
         }
 
         function createResponse() {
           // If no sessionToken, this could be a legacy client
           // attempting to change password, return legacy response.
           if (!sessionTokenId) {
-            return {};
+            return {}
```

Comment:
I'm starting to suspect that we don't run a linter on auth-server.

Reply:
I did turn off prettier for this one, so that it wouldn't auto format things on save and make these changes appear bigger than they actually were. See comment [here](https://github.com/mozilla/fxa/blob/365d747e7223d02cc3f4976c86a37d9e47fb0ed5/packages/fxa-auth-server/test/remote/account_create_tests.js#L21). 

I was sorta of assuming anything super egregious would be caught by the linter though... I was also thinking that after code review prettying the files with changes would be a good call.